ee/clickhouse/queries/trends/total_volume.py

```diff
@@ -61,11 +61,23 @@ def _total_volume_query(self, entity: Entity, filter: Filter, team: Team) -> Tup
                 )
             elif filter.display == TRENDS_CUMULATIVE and entity.math == "dau":
                 cumulative_sql = CUMULATIVE_SQL.format(event_query=event_query)
-                content_sql = VOLUME_SQL.format(event_query=cumulative_sql, **content_sql_params)
+                content_sql = VOLUME_SQL.format(
+                    event_query=cumulative_sql,
+                    start_of_week_fix="0," if filter.interval == "week" else "",
```

Comment:
This is reused very heavily. Could be helpful dropping a util somewhere to handle this in one place

Reply:
Done